app/controllers/collaborations_controller.rb

```diff
@@ -1,2 +1,20 @@
 class CollaborationsController < ApplicationController
+  def create
```

Comment:
You are right. I put it in the application/controller

On Wed, Feb 11, 2015 at 12:54 PM, Eliot Sykes <notifications@github.com>
wrote:

> In app/controllers/collaborations_controller.rb
> <https://github.com/LuisMenjivar/wiki/pull/49#discussion_r24516295>:
>
> > @@ -1,2 +1,20 @@
> >  class CollaborationsController < ApplicationController
> > +  def create
>
> CollaborationsController and some of the other controllers don't use before_action
> :authenticate_user!. Is this right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/wiki/pull/49/files#r24516295>.
>